cmd/main.go

```diff
@@ -22,19 +22,24 @@ import (
 const defaultConfigPath = "config/.config.yaml"
 
 func main() {
-	logger := polyzero.NewLogger()
-
 	configPath, err := getConfigPath()
 	if err != nil {
 		log.Fatalf("failed to get config path: %v", err)
 	}
-	logger.Info().Msgf("Starting PATH using config file: %s", configPath)
 
 	config, err := config.LoadGatewayConfigFromYAML(configPath)
 	if err != nil {
 		log.Fatalf("failed to load config: %v", err)
 	}
 
+	loggerOpts := []polylog.LoggerOption{
+		polyzero.WithLevel(polyzero.ParseLevel(config.Logger.Level)),
+	}
+
+	logger := polyzero.NewLogger(loggerOpts...)
+
+	logger.Info().Msgf("Starting PATH using config file: %s", configPath)
```

Comment:
Would you also be able to add a log indicating the log level (might have to use `fmt` in order for it to always log).

Reply:
@okdas ?